### PR TITLE
Feature/3625  remove workspace level actions in webdav

### PR DESCRIPTION
**Pull request by inkhey**
related to #3625 

- remove workspace level actions in webdav
- refactoring webdav code a bit : remove dead code, reorganize to make things more clear.

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

**Comment by inkhey** [comment]
To test this:
- Try create/rename space from webdav
- Try all webdav action: move/copy/create/update(override content) and verify if no error/ incorrect behavior appears, copy/move have multiple case already tested (depending if we move intra or between multiple space) but it may be better to check this too.

File: backend/doc/known_issues.md [hunk 1]
```diff
@@ -18,3 +18,19 @@ These variables should contain absolute paths and should be quoted.
...
+with same name at the same level through webdav. The webdav view will only show you the "first" one
+according to the tracim database. All the visible spaces are fully working.
+In order to workaround this issue, you can either:
+
+- rename space
+- set same name space as children of different spaces
+
+Be careful about the 2nd solution, user should be members of both parent and children to
+see the hierarchy. If user has only access to 2 children same name space, it will get
+the same issue in webdav.
```
Comment (raphj) [inline backend/doc/known_issues.md:36]:
```suggestion
## WebDAV

### Spaces with the same name are not properly displayed in WebDAV

In Tracim v3.2+. you cannot show multiple spaces
with same name at the same level through WebDAV. The WebDAV view will only show you the one that was created first.
In order to workaround this issue, we recommand that you rename the conflicting spaces.
If it is not an option, you may also move the conflicting spaces into other spaces.
Be careful with this solution: users should be members of the parents to see the full hierarchy. If a user only has access to the spaces with the same name but not their parents, they will still have the issue.
```

**Review by raphj — APPROVED** [review]

**Comment by PhilippeAccorsi** [comment]
Tested on branch with Debian 10 and Windows10 webdav access.

**Comment by raphj** [comment]
The failing test is a Cypress test known to be unstable.

**Pull request merged**